sim/dex.ts

```diff
@@ -826,6 +826,22 @@ export class ModdedDex {
 				ruleTable.set(ruleSpec, '');
 				continue;
 			}
+			if (ruleSpec.split(":").length > 1) {
+				const parts = ruleSpec.split(":");
+				if (parts[0] === 'teamlength') {
+					const vals = ruleSpec.split(":")[1].split('-');
+					const onValidate = parseInt(vals[1]);
+					const onBattle = parseInt(vals[0]);
+					format.teamLength = {
+						validate: [onBattle, onValidate],
+						battle: onBattle
+					}
+				}
+				else if (parts[0] === 'level') {
+					format.maxLevel = parseInt(parts[1]);
```

Comment:
It might be useful to consider `maxForcedLevel` and `forcedLevel`. These are usually what you want for VGC-likes.

Reply:
Hmm but that conflicts with trying to do past generation LC, where maxLevel is what you're looking for. A compromise would be to separate the rules but I think the average PS staff member wouldn't be able to figure out the difference, so you lose intuitiveness there